packages/engine/errors/2693.md

```diff
@@ -0,0 +1,21 @@
+---
+original: "'{0}' only refers to a type, but is being used as a value here."
+excerpt: "I was expecting A as a type declaration but it is being used as a value here."
+---
+
+I think you meant to assign a type to your declaration. So instead of,
```

Comment:
Happy with this as an initial stab! I think this error can be more flexible than this, but this is certainly good for now!

Reply:
Totally! Will keep updating!